### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#43974]
#### VFS team name
Debt Resolution

#### VFS product name
Combined VA Debt Portal

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
When viewing details on a debt (/summary/debt-balances/details/[number]) and there is an alert at the top of the page, the headings skip from `h1` to `h3` with no `h2` between them. Many screen reader users navigate by heading, using the heading levels to explore the structure of the page. It's easy to be disoriented or confused when heading levels are skipped or improperly nested.

### Link, screenshot or steps to recreate
When logged in as test user 81, navigate to [Your Post-9/11 GI Bill debt for housing](https://staging.va.gov/manage-debt-and-bills/summary/debt-balances/details/79612301872). Inspect the code to see that "We're offsetting your benefit payments each month until your debt is paid" is an `h3`.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

---

### Platform Recommendation
The heading on the alert should be an `h2`, not an `h3`.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
